app/src/main/java/ml/docilealligator/infinityforreddit/adapters/RulesRecyclerViewAdapter.java

```diff
@@ -143,18 +165,41 @@ class RuleViewHolder extends RecyclerView.ViewHolder {
         @BindView(R.id.short_name_text_view_item_rule)
         TextView shortNameTextView;
         @BindView(R.id.description_markwon_view_item_rule)
-        TextView descriptionMarkwonView;
+        RecyclerView descriptionMarkwonView;
+        @NonNull
+        final MarkwonAdapter markwonAdapter;
 
         RuleViewHolder(@NonNull View itemView) {
             super(itemView);
             ButterKnife.bind(this, itemView);
             shortNameTextView.setTextColor(mPrimaryTextColor);
-            descriptionMarkwonView.setTextColor(mSecondaryTextColor);
```

Comment:
This did nothing because of `textView.setTextColor(mPrimaryTextColor);` in `beforeSetText` above. I've removed `mSecondaryTextColor` altogether.